### ISSUE TITLE
Plugin install error: Installation failed: The package could not be installed

### ISSUE THREAD

**Issue by seojacky** [afragen/git-updater#977]
Can't install plugins if Git Updater plugin is activated (Version 10.6.9)
<img src="https://i.imgur.com/nhCPxQK.jpg">

**Comment by afragen** [comment]
You must have a plugin conflict. Try disabling plugins until You can figure out where the conflict is. Once you have that narrowed down, please let me know so I can investigate it. 

I can assure you that Git Updater does not conflict with the core plugin installation or updating functionality.

**Comment by seojacky** [comment]
Hmmm..... But after I deactivated Git Updater, I was able to install the plugin? I have checked 3 times. Every time I deactivated Git Updater everything started working correctly ...
I'll check it again.

**Comment by afragen** [comment]
I say this because in 8 yrs I've never had the Git Updater interfere like that. 

It might be worthwhile to bring the site down to a local environment for testing.

**Comment by seojacky** [comment]
I figured out what the problem is. One of the plugins I took from the github had the  line spelled incorrectly `GitHub Plugin URI: https://github.com`
The path was specified not to the repository with the plugin, but simply to the github! After I fixed the line in this plugin, it worked!
Can I advise you to check in the plugin for the existing path to the repository? And if there is an error, then display a message.
